packages/flutter/lib/src/widgets/image.dart

```diff
@@ -90,6 +91,23 @@ class Image extends StatefulWidget {
   }) : image = new NetworkImage(src, scale: scale),
        super(key: key);
 
+  /// Creates a widget that displays an [ImageStream] obtained from a [File].
+  ///
+  /// The [file], [scale], and [repeat] arguments must not be null.
+  Image.file(File file, {
```

Comment:
Can we add a note up in the class-level docs as well? The class-level docs mention the other constructors by name.